.github/workflows/push.yaml

```diff
@@ -73,13 +82,17 @@ jobs:
           password: ${{ secrets.QUAY_TOKEN }}
 
       - name: Push images
-        run: docker push -a ${{ matrix.image_base }}
+        run: docker push -a "${{ steps.img.outputs.base }}"
 
   build-kubesaw:
     name: Build kubesaw components
     runs-on: ubuntu-22.04
     if: ${{ github.event_name == 'pull_request_target' }}
 
+    env:
+      IMAGE_BUILDER: docker
```

Comment:
@sadlerap is this something we want to have?

```suggestion
```

Reply:
We could probably move this to the "Build and push images" step of this job, since that's the only place we need this env var set.

Reply:
done in d6d5a7b3f664fdc8372843896168366d42ab7337